Source/WebCore/css/CSSValue.h

```diff
@@ -292,7 +294,7 @@ class CSSValue {
 
     // NOTE: This class is non-virtual for memory and performance reasons.
```

Comment:
See this comment. This class intentionally doesn't have a virtual destructor. Ignore the bot that's telling you to make it virtual.